samples/excel/85-preview-apis/range-dependents.yaml

```
@@ -0,0 +1,176 @@
+order: 3
+id: excel-range-dependents
+name: Dependents
+description: This sample shows how to find and highlight the dependents of the currently selected cell. Dependent cells contain formulas that refer to other cells.
+host: EXCEL
+api_set:
+    ExcelApi: BETA (PREVIEW ONLY)
+script:
+    content: |
+        $("#setup").click(() => tryCatch(setup));
+        $("#select-D3").click(() => tryCatch(selectD3));
+        $("#get-all-dependents").click(() => tryCatch(getAllDependents));
+        $("#clear-highlighting").click(() => tryCatch(clearFormatting));
+
+        async function selectD3() {
+          await Excel.run(async (context) => {
+            // This method selects a cell with dependents across both worksheets.
+
+            // Activate the sample worksheet.
+            const sheet = context.workbook.worksheets.getItem("Sample2019Data");
+            sheet.activate();
+
+            // Select cell D3.
+            const range = sheet.getRange("D3");
+            range.select();
+            await context.sync();
+          });
+        }
+
+        async function getAllDependents() {
+          await Excel.run(async (context) => {
+            // This method highlights all the dependent cells of the active cell.
+            // Dependent cells contain formulas that refer to other cells.
+
+            // Get the active cell, retrieve the dependent cells, and load their addresses.
+            let range = context.workbook.getActiveCell();
+            let dependents = range.getDependents();
+            range.load("address");
+            dependents.areas.load("address");
+            await context.sync();
+
+            console.log(`All dependent cells of ${range.address}:`);
+
+            // Use the dependents API to loop through dependents of the active cell.
+            for (var i = 0; i < dependents.areas.items.length; i++) {
+              // Highlight and print out the address of each dependent cell.
+              dependents.areas.items[i].format.fill.color = "Orange";
+              console.log(`  ${dependents.areas.items[i].address}`);
+            }
+            await context.sync();
+          });
+        }
+
+        async function clearFormatting() {
+          await Excel.run(async (context) => {
+            // This method removes highlighting from dependent cells.
+
+            const sheet1 = context.workbook.worksheets.getItem("Sample2019Data");
+            let range1 = sheet1.getRange("B3:E5");
+            range1.format.fill.clear();
+
+            const sheet2 = context.workbook.worksheets.getItem("Sample2020Data");
+            let range2 = sheet2.getRange("B3:F5");
```

Favor `const` where applicable.
```suggestion
            const range1 = sheet1.getRange("B3:E5");
            range1.format.fill.clear();

            const sheet2 = context.workbook.worksheets.getItem("Sample2020Data");
            const range2 = sheet2.getRange("B3:F5");
```